upstream-sha256: 525fc067b818f71974bfacfb05152eed14094b26eac5fa448af889511893b177
actual-sha256: e7fc9f0324a36446ccca6134830bd6e75d734cd662638dd394bb0d5e2fb873d6
--- upstream.expected
+++ tolk.actual
@@ -17,8 +17,9 @@
       *(buf0+0) = ((*(buf0+0))+val0);
     }
     float val1 = (*(buf0+0));
-    float alu4 = ((val1<0.0f)?0.0f:val1);
-    *(buf1+0) = ((*(buf1+0))+alu4);
+    float alu4 = ((val1!=val1)?val1:0.0f);
+    float alu5 = ((0.0f<val1)?val1:alu4);
+    *(buf1+0) = ((*(buf1+0))+alu5);
   }
   *(buf3+lidx0) = (*(buf1+0));
   __syncthreads();
